includes/components/class-attribute.php

```diff
@@ -734,6 +740,15 @@ public function add_field_settings( $meta_box ) {
 						'html'       => true,
 						'_order'     => 120,
 					];
+
+					if ( 'user' === $model && 'attachment_upload' !== $field_type ) {
```

Comment:
Just make sure that all the attribute types work in the registration form, I assume issues with the attachments for sure but idk about other types, test some specific ones like Select (single and multiple), Checkboxes